cpp/src/arrow/flight/serialization-internal.cc

```diff
@@ -32,6 +32,214 @@ bool ReadBytesZeroCopy(const std::shared_ptr<arrow::Buffer>& source_data,
   return input->Skip(static_cast<int>(length));
 }
 
+using google::protobuf::io::CodedInputStream;
+using google::protobuf::io::CodedOutputStream;
+
+bool ReadBytesZeroCopy(const std::shared_ptr<arrow::Buffer>& source_data,
```

Comment:
Oh, and you needn't redeclare this (already defined above).

Reply:
done